System/CLI/encryption.py

```diff
@@ -0,0 +1,43 @@
+import KeyUtils
+import hashlib

```

Comment:
Can we use the builtin python hashing library? I've found that hashlib won't install on the development library.